Jenkinsfile.integration

```diff
@@ -90,11 +90,16 @@ pipeline {
                     }
                 }
             }
-        }
-        always {
             script {
-                sh './run.sh teardown'
+                sh './run.sh gather_logs'
```

Comment:
no logs if everything is fine?
I would gather logs before cleanup.

Reply:
Why gather logs if everything is fine?

Reply:
Because of comparisons or trends?
I don't see a reason why logs shouldn't be gathered in fact. Is storage really a problem?